### PR TITLE
[M] CANDLEPIN-871: Added CS8 container to CI

### PR DESCRIPTION
**Pull request by bakajstep**

### CODE REVIEW

File: containers/cs8.Containerfile [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+COPY ${WAR_FILE} ./candlepin.war
+
+# Prepare development certs
+RUN mkdir -p /app/certs
+WORKDIR /app/certs
+COPY ./bin/deployment/gen_certs.sh .
+RUN ./gen_certs.sh --cert_out ./candlepin-ca.crt --key_out ./candlepin-ca.key --hostname candlepin; \
+    rm gen_certs.sh;
+
+
```
Comment (joshmalbrecht) [inline containers/cs8.Containerfile:34]: Extra new line

**Review by joshmalbrecht — CHANGES_REQUESTED** [review]
I just noticed an extra new line in the new containerfile.

I'm also wondering if this containerfile should live in `.github/containers` as it seems that this containerfile is to only be used for spec tests that use the Postgres database.

File: containers/cs8.Containerfile (hunk 1 above)
Comment (bakajstep) [reply containers/cs8.Containerfile]: done

**Comment by bakajstep** [comment]
> I just noticed an extra new line in the new containerfile.
> 
> I'm also wondering if this containerfile should live in `.github/containers` as it seems that this containerfile is to only be used for spec tests that use the Postgres database.

Thats a good question. I was thinking about that too. We can have containers on one place in `containers` folder or have one in `containers` and other in `.github/containers`, but then we would have `Containerfiles` for CI on two diffrent places. Maybe @nikosmoum what dou you think about that?

**Comment by nikosmoum** [comment]
I think moving the cs8 containerfile to `.github/containers` make sense, since the main `/containers` directory is for containers that end up in shippable images (to quay). So it makes sens to keep them in separate places

**Comment by bakajstep** [comment]
Moved to `./.github/containers/`

**Review by joshmalbrecht — APPROVED** [review]

**Pull request merged**